test/specs/test.spec.js

```diff
@@ -0,0 +1,96 @@
+// https://the-internet.herokuapp.com/ - recomended app
```

Comment:
Pls use the same file naming for all files. Look at these files:
- test/specs/test.e2e.js
- test/specs/test.spec.js